### PR TITLE
QUBO to Ising Hamiltonian

### PR DESCRIPTION
**Pull request by Bagherpoor**
A class which converts Qubo matrices (Linear and Quadratic terms) to an Ising Hamiltonian along with an example were added!

### CODE REVIEW

File: tutorials/Enable Session/Sampler_SessionEnable.m
Comment (pacomf) [inline tutorials/Enable Session/Sampler_SessionEnable.m:1]: Remove this change :)

File: tutorials/Qubo_To_IsingHamiltonian/QUBO_Ising.m
Comment (pacomf) [inline tutorials/Qubo_To_IsingHamiltonian/QUBO_Ising.m:1]: Can we add a better structure to this file? With some extra comments about what are the things that you are doing, etc :)

**Review by pacomf — CHANGES_REQUESTED** [review]

File: tutorials/Qubo_To_IsingHamiltonian/QUBO_Ising.m
Comment (Bagherpoor) [reply tutorials/Qubo_To_IsingHamiltonian/QUBO_Ising.m:1]: Sure, I will add more comments to the code! I did it quickly to make sure it works! :)

File: tutorials/Qubo_To_IsingHamiltonian/QUBO_Ising.m [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+
+%% Example of travelling salesman problem and converting it to a Qubo format
+function QP = tsp2qubo(dist)
+%   QP = TSP2QUBO(DIST) returns a QUBO problem from the traveling salesperson
+%   problem specified by the distance matrix DIST. DIST is an N-by-N
+%   nonnegative matrix where DIST(i,j) is the distance between locations
+%   i and j.
+
+% Copyright 2023 The MathWorks, Inc.
```
Comment (pacomf) [inline tutorials/Qubo_To_IsingHamiltonian/QUBO_Ising.m:34]: Why this copyright here?

**Review by pacomf — CHANGES_REQUESTED** [review]

**Review by pacomf — APPROVED** [review]

**Pull request merged**